types/maxmsp/index.d.ts

```diff
@@ -1219,6 +1223,69 @@ declare class Maxobj {
      * others.
      */
     understands(message: string): boolean;
+
+    /**
+     * Returns an Array value containing the names of available attributes for the object.
+     */
+    getattrnames(): string[];
+
+    /**
+     * Returns the value of the attribute specified by attribute_name. Lists are returned as JS Array objects.
+     */
+    getattr(attribute_name: "assistshowspatchername" | "bottomtoolbarpinned" | "boxanimatetime" | "enablehscroll"
```

Comment:
this one somehow didn't show up on my system Max8.3.3.

Added it.

Reply:
Again Pete made me realize that "enabletransparentbgwithtitlebar" is for Max 8.5.0 or later only, I am not sure how we should deal with this kind of versioning stuff in the types.

Do you have any thoughts on this?

Reply:
Personally I'd always be targeting the newest version of Max as there's no real reason not to update any minor version imo. Do you have any thoughts either way on this?

Reply:
As far as I understand it currently the js object will still be in max 9 so I think we will be safe to maintain a single package for this.

I'd still say it's best to target the most current version of max 8 in these definitions and because we can't use the `typesVersions` field in a sensible way when building for max perhaps we just add some comments explaining that certain things are only version > 8.x.x ? Then at least people could see from the definitions why something might not be available in their older version of max. WDYT?

Reply:
Makes sense, let’s do  like that

Reply:
added a comment with the latest push